### PR TITLE
fix(web): fixes error on tab, enter when only a single input element is on the page

### PR DESCRIPTION
**Pull request by jahorton**
Resolves an error reported by a user from KeymanWeb 16.0.

```
domManager.ts:1323 Uncaught TypeError: Cannot read properties of undefined (reading 'focus')
    at b.moveToNext (domManager.ts:1323:7)
    at a.newlineHandler (input.ts:168:11)
    at a.handleNewlineAtCaret (input.ts:154:7)
    at e.defaultRuleBehavior (keyboardProcessor.ts:151:15)
    at e.processKeystroke (keyboardProcessor.ts:277:31)
    at e._processKeyEvent (inputProcessor.ts:166:26)
    at e.processKeyEvent (inputProcessor.ts:101:16)
    at e.handleClick (preProcessor.ts:66:22)
    at e.raiseKeyEvent (preProcessor.ts:44:22)
    at e.modelKeyClick (visualKeyboard.ts:1004:7)
```

It was reported to occur whenever a page contained only one valid context-hosting element - an `<input>` - and when the user pressed the ENTER key on the touch OSK.

@keymanapp-test-bot skip

### CODE REVIEW

File: web/src/app/browser/src/defaultBrowserRules.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default class DefaultBrowserRules extends DefaultRules {
       const contextManager = this.contextManager;
       const activeElement = contextManager.activeTarget?.getElement();
       const nextElement = contextManager.page.findNeighboringInput(activeElement, back);
```
Comment (jahorton) [inline web/src/app/browser/src/defaultBrowserRules.ts:40]: The function can explicitly return `null`, yet we weren't null-guarding on the result.

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.45-alpha](https://keyman.com/downloads/releases/alpha/18.0.45)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.45-alpha](https://keyman.com/downloads/releases/alpha/18.0.45)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.45-alpha](https://keyman.com/downloads/releases/alpha/18.0.45)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.45-alpha](https://keyman.com/downloads/releases/alpha/18.0.45)

**Pull request merged**